frontend/src/components/PdfPreview.js

```diff
@@ -0,0 +1,75 @@
+import React, { Component } from 'react';
+import { Document, Page } from 'react-pdf/dist/entry.noworker';
```

Comment:
Why not use the `react-pdf/dist/entry.webpack'` version, with the workers?

Reply:
but wait, why are you speaking english on this repository ?

Reply:
because it doesn't work, by the way, and i don't have time to debug it